react-accordion - Daniel Yount

TheDanYount opened the pull request:
![Recording 2024-08-30 at 22 22 58](https://github.com/user-attachments/assets/2be41f99-5a0c-4c32-8619-d33bd172ccb9)

TheDanYount committed:
Completed the exercise

RobertGardner commented on react-accordion/my-app/src/Accordion.tsx:
```diff
@@ -0,0 +1,33 @@
+};
+
+export function Accordion({ topics }: Props) {
+  const [currentTopic, setCurrentTopic] = useState('');
```
If you have 2 topics with the same text then this won't work. Topics were given an ID to address this. You should use the ID instead of the text.

RobertGardner commented on react-accordion/my-app/src/TopicCard.tsx:
```diff
@@ -0,0 +1,29 @@
+      </h2>
+      <p
+        className={`my-[-1px] border-2 border-black ${
+          currentTopic === title ? '' : 'hidden'
```
Since you aren't doing any animations, I think this is easier with conditional rendering instead of using 'hidden'.

TheDanYount committed:
Added suggestions

Used conditional rendering and used id rather than title to account for duplicate titles
